src/Ifera/ScoreHud/session/PlayerManager.php

```diff
@@ -33,15 +33,15 @@
 
 namespace Ifera\ScoreHud\session;
 
-use pocketmine\Player;
+use pocketmine\player\Player;
 
 class PlayerManager{
 
 	/** @var PlayerSession[] */
 	private static $sessions = [];
 
 	public static function create(Player $player): void{
-		self::$sessions[$player->getRawUniqueId()] = $session = new PlayerSession($player);
```

Comment:
`getRawUniqueId()` is superseded by `getUniqueId()->getBytes()`.  

This is seen here: https://github.com/pmmp/PocketMine-MP/blob/master/changelogs/4.0.md?#L499

Reply:
I guess its fine for now, I will most likely be rewriting parts of ScoreHud and update it accordingly.